### PR TITLE
Test enhancement

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log
- Defining multiple `PHPUnit` versions to support different `PHP` versions because of compatibility.
- Add the `-n` option to ignore warning message output during `phpcs` coding style check.
- Using the class-based namespace to be compatible with all `PHPUnit` verions.
- Remove the `@expectedExceptionMessage` annotation because every `cURL` version have different error message.
- The `MaxMind\\WebService\\Http\\Request` class constructor has the arguments and it adds missing arguments.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,7 +27,7 @@ install:
 script:
   - mkdir -p build/logs
   - "if [[ $RUN_LINTER ]]; then vendor/bin/php-cs-fixer fix --verbose --diff --dry-run --config=.php_cs; fi"
-  - "if [[ $RUN_LINTER ]]; then vendor/bin/phpcs --standard=PSR2 src/; fi"
+  - "if [[ $RUN_LINTER ]]; then vendor/bin/phpcs -n --standard=PSR2 src/; fi"
```
Comment (oschwald) [inline .travis.yml]: Again, we want the warnings and we don't want the whitespace changes to make very long lines.

File: tests/MaxMind/Test/WebService/Http/CurlRequestTest.php [hunk 2]
```diff
@@ -40,7 +40,6 @@ public function testGet()
 
     /**
      * @expectedException \MaxMind\Exception\HttpException
-     * @expectedExceptionMessage cURL error (6):
```
Comment (oschwald) [inline tests/MaxMind/Test/WebService/Http/CurlRequestTest.php]: Why was this removed?

File: .travis.yml (hunk 1 above)
Comment (peter279k) [reply .travis.yml]: Hi @oschwald, thanks for your reply.

If we set the `-n` to ignore warning message, the `PHP_CodeSniffer` will be passed.

And the `PHP-CS-Fixer` will be also passed.

We remove `-n` from `PHP_CodeSniifer` coding style check and fix them.

Then the `PHP-CS-Fixer` will have the problematic coding style.

They're conflicted. The related Travis CI build log is as follows:

- The problematic coding style from `PHP-CS-Fixer`.
https://travis-ci.org/open-source-contributions/web-service-common-php/jobs/607750057?utm_medium=notification&utm_source=github_status

File: tests/MaxMind/Test/WebService/Http/CurlRequestTest.php (hunk 2 above)
Comment (peter279k) [reply tests/MaxMind/Test/WebService/Http/CurlRequestTest.php]: Revert this.

File: .travis.yml (hunk 1 above)
Comment (oschwald) [reply .travis.yml]: Aha, I apologize; I did not realize this was a [new rule in php-cs-fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4452). It was only released a few days ago.

I think we could just disable that rule by adding `'single_line_throw' => false,` to `.php_cs`.

Comment (oschwald) [reply .travis.yml]: I disabled that rule on all of the repos. If you rebase, you should get it.

**Comment by peter279k** [comment]
Hi @oschwald, thanks for your help.

I've merged commit from master and revert the coding style.

Please review this PR again. Thanks.

File: tests/MaxMind/Test/WebService/Http/CurlRequestTest.php [hunk 3]
```diff
@@ -26,7 +27,6 @@ class CurlRequestTest extends \PHPUnit_Framework_TestCase
...
```
Comment (oschwald) [inline tests/MaxMind/Test/WebService/Http/CurlRequestTest.php]: Could you add the message back for this one as well?

**Comment by oschwald** [comment]
Thanks!
